src/components/05_pages/NodeForm/index.js

```diff
@@ -77,10 +79,16 @@ class NodeForm extends React.Component {
           .includes(key),
       )
       .reduce((agg, [key, value]) => ({ ...agg, [key]: value }), {});
+
+    // Set default `Authored By` relationship.
+    if (!Object.prototype.hasOwnProperty.call(props, 'entity')) {
```

Comment:
Did you see if this could be added in the `NodeAddForm` component? It would be nice if it's added there so it's clear that it is only applied on the entity creation.

Reply:
Done.